helm/cloud-provider-cloud-director/charts/cloud-director-named-disk-csi-driver/templates/pss-exceptions.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
```

Comment:
@jkremser We don't have any solution for the chicken-egg problem you mentioned. Right? Will we keep this PR until we have a solution?

Reply:
right, this one should rather wait and then got the same solution as the vsphere cloud provider

Reply:
ok, so I am removing the polex from this pr and opened this: https://github.com/giantswarm/kyverno-app/pull/293